backend/kernelCI_app/views/testsByCommitHashView.py

```diff
@@ -39,7 +45,8 @@ def get(self, request, commit_hash: str | None):
                 t.environment_comment, t.misc FROM checkouts AS c
                 INNER JOIN builds AS b ON c.id = b.checkout_id
                 INNER JOIN tests AS t ON t.build_id = b.id
-                WHERE c.git_commit_hash = %s {path_filter}
+                WHERE c.git_commit_hash = %s AND c.origin = %s AND c.git_repository_url = %s
+                  AND c.git_repository_branch = %s {path_filter}
                 ORDER BY
                 build_id,
```

Comment:
we should order by path here

Reply:
yeah, I realize it when I did in #171, but I will adjust it here